bin/prettier.js

```diff
@@ -193,7 +193,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log(chalk.grey("%s %dms"), filename, Date.now() - start);
```

Comment:
Why are you showing the time taken for files that haven't changed? That seems random and not really connected with whether or not they changed.